python/core/additions/qgsfunction.py

```diff
@@ -185,7 +185,7 @@ def qgsfunction(args="auto", group="custom", **kwargs):
         * *usesgeometry* (``bool``) --
           Defines if this expression requires the geometry. By default False.
         * *referenced_columns* (``list``) --
-          An array of field names on which this expression works. By default ``[QgsFeatureRequest.ALL_ATTRIBUTES]``. Can be set to None for slightly faster evaluation.
+          An array of field names on which this expression works. By default ``[QgsFeatureRequest.ALL_ATTRIBUTES]``. Can be set to an empty list for slightly faster evaluation.
```

Comment:
@gacarrillor should the two texts be the same?

Reply:
Done :+1: